LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 PromLabs GmbH
```

Comment:
This line is part of the instructions on how to use the license (e.g. in code files), not the actual copyright line for this library itself. We had the same situation in Prometheus :)

So far I haven't put the license text at the top of all files of the repo yet, I guess we could do that at some point.

Reply:
mmm ok. Yeah I guess we can add the license on top of each file